x-pack/filebeat/input/s3/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package s3
+
+import "github.com/elastic/beats/filebeat/harvester"
+
+var defaultConfig = config{
+	ForwarderConfig: harvester.ForwarderConfig{
+		Type: "s3",
+	},
+}
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	QueueURLs                 []string `config:"queueURLs"`
```

Comment:
Is this given to you in AWS UI for copy/paste?

Reply:
Yes! 
<img width="782" alt="Screen Shot 2019-06-25 at 8 31 21 AM" src="https://user-images.githubusercontent.com/14081635/60107191-c10cc580-9723-11e9-852e-a06dbb8dfd50.png">